timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-13 14:56:13,ERROR,demo13,create_or_pass_folder,202,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:13,ERROR,demo13,create_or_pass_folder,202,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:13,ERROR,demo13,create_or_pass_folder,211,0,"Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:22b91af9-801e-004e-3873-bdb0e3000000
Time:2024-06-13T09:26:13.4820282Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:13,ERROR,demo13,create_or_pass_folder,211,0,"Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951a6d5d-201e-0007-7873-bd8383000000
Time:2024-06-13T09:26:13.4812611Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:13,INFO,demo13,home,1411,1,User demo13 logged in successfully,
2024-06-13 14:56:13,INFO,demo13,home,1411,1,User demo13 logged in successfully,
2024-06-13 14:56:14,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 14:56:14,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 14:56:14,ERROR,demo13,create_or_pass_folder,202,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:14,ERROR,demo13,create_or_pass_folder,211,0,"Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:ba7be069-801e-0001-7173-bd74fb000000
Time:2024-06-13T09:26:14.8373770Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:56:14,INFO,demo13,home,1411,1,User demo13 logged in successfully,
2024-06-13 14:56:15,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 14:56:17,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 14:56:17,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 14:56:25,INFO,demo13,get_data_source,2150,1,table_update route successfully send data,
2024-06-13 14:59:58,ERROR,demo13,create_or_pass_folder,202,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:59:58,ERROR,demo13,create_or_pass_folder,211,0,"Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'demo13': Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\app.py"", line 195, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\Main_CogniLink_App\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:demo13
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:951bb38f-201e-0007-7374-bd8383000000
Time:2024-06-13T09:29:58.6684980Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>demo13</QueryParameterValue></Error>"
2024-06-13 14:59:58,INFO,demo13,home,1411,1,User demo13 logged in successfully,
2024-06-13 14:59:59,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 15:00:02,INFO,demo13,update_bar_chart_from_blob,323,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-13 15:00:10,INFO,demo13,get_data_source,2150,1,table_update route successfully send data,
